test/algorithms/Jamfile

```diff
@@ -16,56 +16,53 @@
-    [ run num_interior_rings_multi.cpp : : : : algorithms_num_interior_rings_multi ]
-    [ run num_points.cpp               : : : : algorithms_num_points ]
-    [ run num_points_multi.cpp         : : : : algorithms_num_points_multi ]
-    [ run num_segments.cpp             : : : : algorithms_segments ]
-    [ run perimeter.cpp                : : : : algorithms_perimeter ]
-    [ run perimeter_multi.cpp          : : : : algorithms_perimeter_multi ]
-    [ run point_on_surface.cpp         : : : : algorithms_point_on_surface ]
-    [ run remove_spikes.cpp            : : : : algorithms_remove_spikes ]
-    [ run reverse.cpp                  : : : : algorithms_reverse ]
-    [ run reverse_multi.cpp            : : : : algorithms_reverse_multi ]
-    [ run simplify.cpp                 : : : : algorithms_simplify ]
-    [ run simplify_multi.cpp           : : : : algorithms_simplify_multi ]
-    [ run transform.cpp                : : : : algorithms_transform ]
-    [ run transform_multi.cpp          : : : : algorithms_transform_multi ]
-    [ run unique.cpp                   : : : : algorithms_unique ]
-    [ run unique_multi.cpp             : : : : algorithms_unique_multi ]
-    ;
+run append.cpp                   : : : : algorithms_append ;
+run assign.cpp                   : : : : algorithms_assign ;
+run azimuth.cpp                  : : : : algorithms_azimuth ;
+run centroid.cpp                 : : : : algorithms_centroid ;
+run centroid_multi.cpp           : : : : algorithms_centroid_multi ;
+run comparable_distance.cpp      : : : : algorithms_comparable_distance ;
+run convert.cpp                  : : : : algorithms_convert ;
+run convert_multi.cpp            : : : : algorithms_convert_multi ;
+run correct.cpp                  : : : : algorithms_correct ;
+run correct_multi.cpp            : : : : algorithms_correct_multi ;
+run correct_closure.cpp          : : : : algorithms_correct_closure ;
+run densify.cpp                  : : : : algorithms_densify ;
+run for_each.cpp                 : : : : algorithms_for_each ;
+run for_each_multi.cpp           : : : : algorithms_for_each_multi ;
+run is_convex.cpp                : : : : algorithms_is_convex ;
+run is_empty.cpp                 : : : : algorithms_is_empty ;
+run is_simple.cpp                : : : : algorithms_is_simple ;
+run is_simple_geo.cpp            : : : : algorithms_is_simple_geo ;
+run is_valid.cpp                 : : : : algorithms_is_valid ;
+run is_valid_failure.cpp         : : : : algorithms_is_valid_failure ;
+run is_valid_geo.cpp             : : : : algorithms_is_valid_geo ;
+run is_valid.cpp                 : : : <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_is_valid_alternative ;
+run is_valid_failure.cpp         : : : <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_is_valid_failure_alternative ;
+run is_valid_geo.cpp             : : : <define>BOOST_GEOMETRY_ROBUSTNESS_ALTERNATIVE : algorithms_is_valid_geo_alternative ;
```

Comment:
In the meantime, this is gone and should be omitted.
What are the plans with this PR?